src/gatling/scala/uk/gov/hmcts/reform/sendletter/EmbeddedPdfsSimulation.scala

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.sendletter
+
+import com.typesafe.config.{Config, ConfigFactory}
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
```

Comment:
is there really no other way to write gatling other than "import everything"? especially when code snippet is so small